### ISSUE TITLE
Check that all already implemented UI-features support priorities

### ISSUE THREAD

**Issue by jschummer** [HSLdevcom/jore4#704]
For example, visualizing routes and stops in the different views does not show only the basic route on a specific day (when also a temporary version exists).

Check the priorities everywhere in the UI. The logic should be to show the highest priority in respect to the production. The drafts will be shown elsewhere.

Specify with design.

Blocked by #706

**Comment by eleksi** [comment]
New card created for implementation task.

https://github.com/HSLdevcom/jore4/issues/920
